### ISSUE TITLE
Grouped List accessibility: NVDA + Chrome

### ISSUE THREAD

**Issue by Hardhik** [OfficeDev/office-ui-fabric-react#10169]
### Environment Information

- **Package version(s)**: office-ui-fabric-react@7
- **Browser and OS versions**: Chrome 76.0.3809.100, Win10
- **Screen reader(s)**: NVDA 2016.4

### Describe the issue:

When using NVDA, it reads out the whole of the hierarchy of DOM elements especially noticeable when there's nested Grouped Lists.
For example, in the codepen link provided, when we tab to the button, it reads out
"table  with 0 rows and 0 columns 
text
button
"

instead of just reading
"text
button"

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/hardhik/pen/BaBKgqO

#### Actual behavior:

It reads out
"table  with 0 rows and 0 columns 
text
button
"

#### Expected behavior:
only the current element under focus should be read

### Documentation describing expected behavior

https://www.etsi.org/deliver/etsi_en/301500_301599/301549/02.01.02_60/en_301549v020102p.pdf
WCAG 2.1

**Comment by JasonGore** [comment]
Ok, I did a fair amount of investigation and this problem appears to be multi-faceted.

The design of GroupedList makes this a region of role type `grid` introduced with #7063. This will cause accessibility information to be read out, even when you only have one line of info. Inherently, GroupedList is a group of lists, so I think hearing accessibility information in this instance should be expected when entering the first row of a group.

However, in this case, the row and column info is being read out as 0 rows and 0 columns, so it's not very meaningful.

I think in this case the fix isn't actually to remove the extra accessibility information, but correct it to saw "row 1 column 1". The other option is to support non-table groups, but it's possible that is out of scope for GroupedList's purpose.

@natalieethell @KevinTCoughlin Do you have an opinion as to the action (if any) we could make here?

**Comment by natalieethell** [comment]
@JasonGore I agree that I think the move here is to correct the list to say "1 row 1 column". I believe we need the proper roles, `row` and `gricell`, to achieve this. I made slight changes to the CodePen here that add both of those attributes and the proper roles: https://codepen.io/naethell/pen/gOYwgEo. Narrator + Edge and NVDA + Chrome should now be reading out "1 row 1 column" now, but let me know if you're seeing something different.

If this seems like the proper solution, we should probably update the second example on the GroupedList page to have these additional attributes.

**Comment by natalieethell** [comment]
Jason and I spoke offline a little more and we agree that we might actually want to make some kind of change to the List code itself. We might want to enforce a role on the cell (or row) element so that folks using a custom `onRenderCell` don't have to specify one themselves.

**Comment by micahgodbolt** [comment]
@natalieethell digging into this old issue. Is your last comment the most recent info? Any thoughts as I dig into it?

**Comment by micahgodbolt** [comment]
Does this issue repro in Narrator?

**Comment by natalieethell** [comment]
@micahgodbolt yes, my last comment would be the most recent info.

**Comment by natalieethell** [comment]
I don't believe there are any issues in Narrator + Edge, just NVDA + Chrome.

**Comment by khmakoto** [comment]
@Hardhik, can you confirm what @natalieethell said about this not being an issue with Edge + Narrator? Or does the bug reproduce there as well?

**Comment by Hardhik** [comment]
@khmakoto Narrator + Edge reads it out correctly as "text. Button." So, there's no issue with Edge + Narrator

**Comment by khmakoto** [comment]
Hi @Hardhik, our support matrix is Narrator + Edge, and since it seems we're setting the aria parameters correctly I would suggest raising this as an issue with either NVDA or Chrome. If you do think there is an issue with the way we're setting the aria parameters please let me know.
